src/components/Schedule.jsx

```diff
@@ -1,36 +1,71 @@
 import CommercialAlert from "./CommercialAlert";
 import InActiveRouteAlert from "./InActiveRouteAlert";
-import PropTypes from "prop-types";
 import React from "react";
 import ScheduleTable from "./ScheduleTable";
+import { useParams } from "react-router-dom";
+import { useQuery } from "react-query";
+import { Config } from "@baltimorecounty/javascript-utilities";
+import Fetch from "../common/Fetch";
+import SomethingWentWrongAlert from "./SomethingWentWrongAlert";
+import { FormatAddress } from "../common/Formatters";
+import ResetForm from "./ResetForm";
+
+const { getValue } = Config;
+
+const Schedule = () => {
+  const { address } = useParams();
+
+  const { data, status } = useQuery(
+    address && [
+      "getSchedule",
+      {
+        endpoint: getValue("collectionSchedule"),
+        path: `${address}`,
+      },
+    ],
+    Fetch,
+    {
+      refetchOnWindowFocus: false,
+      retries: false,
+    }
+  );
+
+  if (!data) {
+    return <p>Loading collection schedule for {FormatAddress(address)}...</p>;
+  }
+
+  if (status === "error") {
+    return <SomethingWentWrongAlert />;
+  }
 
-const Schedule = ({ schedule = {} }) => {
   const {
     collectionSchedules = [],
     isSingleFamilyHome,
     isActiveRoute,
     pdfLink,
-  } = schedule;
-
-  if (!isActiveRoute) {
-    return <InActiveRouteAlert />;
-  }
-
-  if (!isSingleFamilyHome) {
-    return <CommercialAlert />;
-  }
+    status: httpStatus,
+  } = data;
 
   const hasAtLeastOneSchedule = collectionSchedules.some(
     (schedule) => schedule.nextCollectionDate
   );
 
   return (
     <div>
-      {hasAtLeastOneSchedule ? (
-        <ScheduleTable collectionSchedules={collectionSchedules} />
+      <div className="results">
+        <h3>Your Schedule</h3>
+        <p>Showing results for:</p>
+        <p className="font-weight-bold">{FormatAddress(address)}</p>
+        <ResetForm />
```

Comment:
I think this should probalby be `ResetFormMessage` or something along those lines.

Reply:
renamed to `WrongAddressMessage`